governance/IP_POLICY_GUIDANCE.md

```diff
@@ -53,15 +59,27 @@ All project websites need to have a [standard website footer][].
 
 Each project must adopt either the [Developer Certificate of Origin (DCO)][DCO] or a Contributor License Agreement (CLA).
 
+#### DCO
+
+There are two options for implementing a DCO on your project:
+
+1. Via installing the [DCO bot](https://github.com/apps/dco) 
+2. By including the following in your repository:
+   1. Adding the DCO to your CONTRIBUTING.md file, [example](https://github.com/nodejs/node/blob/main/CONTRIBUTING.md)
+   2. Adding the DCO to your PR templates, [example](https://raw.githubusercontent.com/nodejs/node/main/.github/PULL_REQUEST_TEMPLATE.md)
+
+The DCO Bot is the preferred method and is mandatory for any new projects joining the foundation. We also recommend this method for existing projects that require stricter enforcement.
```

Comment:
Are there any downsides to using the bot?

If there aren't, I think that's the only option we should be listing.

Reply:
Not everyone trusts bots with permissions. I think it's good to provide an alternative.

Reply:
> The DCO Bot is the preferred method and is mandatory for any new projects joining the foundation

I wouldn't make this mandatory. Besides, this statement conflicts with the one above about *either* adopting a DCO or CLA. If going the CLA route, the DCO bot would not be mandatory.

Reply:
Permissions requested are fairly lightweight:

- Read access to code, metadata, and pull requests
- Read and write access to checks

I'm genuinely curious what the concern is. (Not trying to argue for the bot, just trying to understand.)

Reply:
I’m not concerned about permissions; i don’t want a non-human entity speaking prose to my contributors.

Reply:
Unless something has changed there were concerns with using the DCO bot when it was proposed for the Node.js project a number of years ago. Brian (formerly OpenJS staff) was going to try to PR in changes to address those concerns but I don't think they every landed.

Reply:
The problem with the bot is that most people don't know how to DCO commits, rebase, or do any advanced git tasks. Essentially, it forcefully raises the barrier to contributions. I have the bot enable on Platformatic, and every other contributors can't fix it.